### ISSUE TITLE
UI Panel placement shifted below tab bar on Linux

### ISSUE THREAD

**Issue by eboracus** [Osmose/advanced-open-file#136]
On Linux it seems the placement of the AOF panel when it pops up is shifted down to below the tab bar, instead of starting from the top of the window, see below:

![aof_panel_location](https://cloud.githubusercontent.com/assets/11758299/22230519/feecdf4e-e1d5-11e6-95ff-ced74547914f.png)

This shift down also means that when the displayed directory has many files the panel reaches the bottom of the Atom window rather than leaving a nice gap by that same amount.

**Comment by Osmose** [comment]
Thanks for the report!

That's expected, insofar that One Dark has had that gap for a while (I think, anyway). It's a theme-specific bug, rather than a platform bug (unless you're also seeing that on, say, Atom Dark, which would be much more surprising).

AFAICT there's two options to fixing little styling bugs like this: either AOF maintains theme-specific fixes, or the skins do. Since I'm not bothered by the gap and no one has submitted fixes, AOF doesn't have any, but I've seen a few themes that actually have AOF-specific fixes. A lovely thing of them to do, they're fabulous for it. :D

I'm open to merging theme-specific fixes if anyone puts a PR up. Doesn't seem like a huge deal having them.

**Comment by eboracus** [comment]
Yes you're right, my mistake. 👍 I wasn't seeing it on other platforms side-by-side, but see now that I'd forgotten to update the UI theme (above image is using atom-dark) on that machine to match the others 😕 ..(doesn't look hugely different on glance).

Switched over and problem solved here. :)
